pyaci/core.py

```diff
@@ -161,13 +161,21 @@ def _rootApi(self):
 
 
 class Node(Api):
-    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None):
+    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None, aciMetaFilePath=None):
         super(Node, self).__init__()
         self._url = url
         if session is not None:
             self._session = session
         else:
             self._session = requests.session()
+
+        if aciMetaFilePath:
+            with open(aciMetaFilePath, 'rb') as f:
+                logger.debug('Loading meta information from %s', aciMetaFilePath)
+                aciMetaContents = json.load(f)
+                self._aciClassMetas = aciMetaContents['classes']
+        else:
+            self._aciClassMetas = aciClassMetas
```

Comment:
We need to add an additional check here. If `aciMetaFilePath` is not specified, and the top level `aciClassMetas` is empty dict, we should raise an exception asking the user to specify meta properly.

Reply:
Done !